app/src/main/res/values/strings.xml

```diff
@@ -227,66 +227,66 @@
-    <string name="search_history_summary">Store the search queries locally</string>
+    <string name="data_saver_mode">Data-saver mode</string>
+    <string name="data_saver_mode_summary">Skip thumbnails and other images.</string>
+    <string name="search_history_summary">Remember searches</string>
     <string name="watch_history_summary">Keep track of watched videos locally</string>
     <string name="history_summary">Watch and search history</string>
-    <string name="watch_positions_title">Watch positions</string>
-    <string name="reset_watch_positions">Reset watch positions</string>
+    <string name="watch_positions_title">Remembered playback positions</string>
+    <string name="reset_watch_positions">Reset</string>
     <string name="system_caption_style">System caption style</string>
     <string name="captions">Captions</string>
     <string name="none">None</string>
-    <string name="update_now">Do you want to update the app now?</string>
-    <string name="seekbar_preview">Seekbar preview</string>
-    <string name="seekbar_preview_summary">Preview the video by seeking to the position when scrubbing the seekbar.</string>
+    <string name="update_now">Install the new LibreTube now?</string>
+    <string name="seekbar_preview">Video preview</string>
+    <string name="seekbar_preview_summary">Show a snapshot when dragging the playback indicator.</string>
     <string name="general">General</string>
-    <string name="general_summary">Language, region</string>
-    <string name="playingOnBackground">Playing on background …</string>
-    <string name="caption_settings">Caption settings</string>
-    <string name="downloading_apk">Downloading Apk …</string>
+    <string name="general_summary">Language, and region</string>
+    <string name="playingOnBackground">Playing in the background…</string>
+    <string name="caption_settings">Captions</string>
+    <string name="downloading_apk">Downloading APK…</string>
     <string name="playerAudioFormat">Audio format for player</string>
     <string name="playerAudioQuality">Audio quality</string>
-    <string name="best_quality">Best quality</string>
-    <string name="worst_quality">Worst quality</string>
-    <string name="default_subtitle_language">Default subtitle language</string>
+    <string name="best_quality">Best</string>
+    <string name="worst_quality">Worst</string>
+    <string name="default_subtitle_language">Subtitle language</string>
     <string name="notifications">Notifications</string>
-    <string name="notify_new_streams">New streams notifications</string>
-    <string name="notify_new_streams_summary">Notify when new streams from subscriptions are available</string>
-    <string name="checking_frequency">Checking frequency</string>
-    <string name="new_streams_count">%1$s new streams are available</string>
-    <string name="new_streams_by">New streams by %1$s …</string>
+    <string name="notify_new_streams">Livestream notifications</string>
```

Comment:
It has nothing to do with livestreams, it just notifies for new uploaded videos by channels the user subscribed to